README.md

```diff
@@ -0,0 +1 @@
+Testing project for learning
```

Comment:
[Severity: Low] The README file is very brief and lacks detailed information about the project.
Consider expanding the README to include sections such as project description, installation instructions, usage examples, and contribution guidelines.
Rationale: A comprehensive README helps users and contributors understand the project better and facilitates easier onboarding.
Reference: The GitHub guide on writing a good README recommends including detailed project information.